.github/workflows/check-standard.yaml

```diff
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Are these what you want in terms of triggers? We can always adjust these to something more specific.

Reply:
I switched to run on every push. It's useful, even though it's a little on the slow side. (7 minutes)